spacy_llm/pipeline/llm.py

```diff
@@ -209,22 +215,18 @@ def __init__(
             max_batches_in_mem=int(cache["max_batches_in_mem"]),
             vocab=vocab,
         )
+        if not Doc.has_extension("llm_prompt"):
+            Doc.set_extension("llm_prompt", default=None)
+        if not Doc.has_extension("llm_response"):
+            Doc.set_extension("llm_response", default=None)
```

Comment:
I would move this to `__call__` and `pipe` (repeated in both) so that it works with multiprocessing with spawn.

Reply:
Or I guess possibly `_process_docs` instead, but it definitely needs to be called again in `pipe`.

Reply:
pulling this out into its own PR